### PR TITLE
User authentication added

### PR DESCRIPTION
**Pull request by mjgasior-ASU**
username: "user"
password: "password"

note: in the red, it says I deleted the "new intent" code. I just moved it inside the positive portion of the if statement.

### CODE REVIEW

File: app/src/main/java/webdevils/webdevilsapp/LoginActivity.java [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
 import android.widget.Button;
 import android.widget.EditText;
+import java.util.HashMap;
+import android.content.DialogInterface;
+import android.app.AlertDialog.Builder;
+
 
 public class LoginActivity extends AppCompatActivity {
 
+    HashMap<String, String> users = new HashMap<String, String>();
```
Comment (kjbryan1) [inline app/src/main/java/webdevils/webdevilsapp/LoginActivity.java]: For now we can change...
HashMap<String, String> users = new HashMap<String, String>(); 
to two strings...
String testUserName = "user";
String testPassword = "password";
(you'll have to update your validateUser() method ...) 
or we can change ...
HashMap<String, String> users
to
HashMap<String, String> testUser

**Review by kjbryan1 — COMMENTED** [review]
Hey Martin,

For now we can change...
HashMap<String, String> users = new HashMap<String, String>();
to two strings...
String testUserName = "user";
String testPassword = "password";
(you'll have to update your validateUser() method ...)
or we can change ...
HashMap<String, String> users
to
HashMap<String, String> testUser

Either way, we'll have to update this when I get the server online...

**Comment by mjgasior-ASU** [comment]
Done.

**Comment by kjbryan1** [comment]
Looks good. This version of the app runs fine on my end.

**Pull request merged**